erpnext/patches/v13_0/add_default_project_into_project_details_child_table.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('projects', 'doctype', 'task')
+	frappe.reload_doc('projects', 'doctype', 'project_item')
+
+	tasks = frappe.db.get_all("Task", fields=["*"])
+	for task in tasks:
+		project_item = frappe.db.get_all("Project Item", filters={"parent":task.name}, fields=["*"])
+		if not project_item:
```

Comment:
```suggestion
		project_item = frappe.db.get_all("Project Item", filters={"parent":task.name}, fields=["*"])
		if not project_item:
```
- This check is invalid since we already know that there wont be any items linked in Project Item since we are introducing it now in this PR